src/coreclr/debug/daccess/nidump.h

```diff
@@ -283,10 +283,10 @@ class NativeImageDumper
     IMetaDataAssemblyImport *m_manifestAssemblyImport;
 
     //helper for ComputeMethodFixupHistogram
-    BOOL HandleFixupForHistogram(PTR_CORCOMPILE_IMPORT_SECTION pSection, SIZE_T fixupIndex, SIZE_T *fixupCell);
+    BOOL HandleFixupForHistogram(PTR_CORCOMPILE_IMPORT_SECTION pSection, SIZE_T fixupIndex, SIZE_T *fixupCell, BOOL notUsed = TRUE);
 
     //helper for DumpMethodFixups
-    BOOL HandleFixupForMethodDump(PTR_CORCOMPILE_IMPORT_SECTION pSection, SIZE_T fixupIndex, SIZE_T *fixupCell);
+    BOOL HandleFixupForMethodDump(PTR_CORCOMPILE_IMPORT_SECTION pSection, SIZE_T fixupIndex, SIZE_T *fixupCell, BOOL notUsed = TRUE);
```

Comment:
These new parameters are indeed not used in `NativeImageDumper::HandleFixupForHistogram`, `NativeImageDumper::HandleFixupForMethodDump`. They are required, because `Module::FixupDelayListAux` should pass this flag to `Module::FixupNativeEntry`, and `Module::FixupDelayListAux` is used for all three to traverse over fixups.

Reply:
Oh ok, it would be good to give it the same name so that it's clear what that parameter is a placeholder for, even if unused